servicetest/timingprofiling/test_profiling.py

```diff
@@ -165,7 +165,7 @@ def get_log_point(log_file, point_name, match_number=1):
             match = match + 1
             if match == match_number:
                 return remove_ansi(string.split(line)[4])
-    print "Nothing found! " + point_name + " " + str(match_number)
+    print("Nothing found! " + point_name + " " + str(match_number))
```

Comment:
Use .format instead of +
https://softwareengineering.stackexchange.com/questions/304445/why-is-s-better-than-for-concatenation